packages/blade/src/components/ButtonGroup/StyledButtonGroup.tsx

```diff
@@ -0,0 +1,36 @@
+import styled from 'styled-components';
+import type { StyledButtonGroupProps } from './types';
+import BaseBox from '~components/Box/BaseBox';
+import { makeBorderSize } from '~utils';
+import { getBackgroundColorToken } from '~components/Button/BaseButton/BaseButton';
+import getIn from '~utils/lodashButBetter/get';
+
+const StyledButtonGroup = styled(BaseBox)<StyledButtonGroupProps>(
+  ({ theme, isDisabled, variant = 'primary', color, isFullWidth }) => {
+    return {
+      display: 'flex',
+      width: isFullWidth ? '100%' : 'fit-content',
+      borderRadius: makeBorderSize(theme.border.radius.medium),
+      borderWidth: makeBorderSize(theme.border.width.thin),
+      borderColor: getIn(
+        theme.colors,
+        getBackgroundColorToken({
+          property: variant === 'tertiary' ? 'background' : 'border',
```

Comment:
why is tertiary button using `background` token for border? can you check with designer once?

Reply:
we need it for secondary variant only, fixed.